src/hotspot/share/opto/superword.cpp

```diff
@@ -1562,90 +1569,135 @@ void SuperWord::combine_packs() {
     }
   }
 
-  // Split packs which have size greater then max vector size.
-  for (int i = 0; i < _packset.length(); i++) {
-    Node_List* p1 = _packset.at(i);
-    if (p1 != nullptr) {
-      uint max_vlen = max_vector_size_in_def_use_chain(p1->at(0)); // Max elements in vector
-      assert(is_power_of_2(max_vlen), "sanity");
-      uint psize = p1->size();
-      if (!is_power_of_2(psize)) {
-        // We currently only support power-of-2 sizes for vectors.
+  // Remove all nullptr from packset
+  compress_packset();
```

Comment:
Note: I will refactor `combine_packs` so that it does not need this in the future. But for now the "combine" code does introduce `nullptr`, and I want to remove them here, to establish the new invariant: no `nullptr` in packlist.